homeassistant/components/volkswagencarnet/__init__.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+import logging
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+
+from datetime import timedelta
+from homeassistant.const import (CONF_USERNAME, CONF_PASSWORD, CONF_RESOURCES, CONF_NAME, CONF_SCAN_INTERVAL)
+from homeassistant.helpers import discovery
+from homeassistant.helpers.event import track_point_in_utc_time
+from homeassistant.util.dt import utcnow
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.dispatcher import dispatcher_send
+from homeassistant.helpers.icon import icon_for_battery_level
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'volkswagencarnet'
+DATA_KEY = DOMAIN
+CONF_REGION = 'region'
+DEFAULT_REGION = 'SV'
+CONF_MUTABLE = 'mutable'
+
+REQUIREMENTS = ['volkswagencarnet==4.0.20']
+
+SIGNAL_STATE_UPDATED = '{}.updated'.format(DOMAIN)
+
+MIN_UPDATE_INTERVAL = timedelta(minutes=1)
+DEFAULT_UPDATE_INTERVAL = timedelta(minutes=5)
+
+COMPONENTS = {
```

Comment:
So uuh, this map always maps to the same thing?
Does that mean it can be removed?

Reply:
It's good to define the map here of its used to couple the two APIs correctly. We don't want to have an implicit coupling of the library API and the home assistant API, which could break when an API changes.